### PR TITLE
ci: separate workflow for specification tests

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
removed the invalid PR comment thing for netlify as cancel isn't really a break (it is config that we don't deploy unless the site changes)

File: .github/workflows/spectest.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: WebAssembly Specification Test
```
Comment (codefromthecrypt) [inline .github/workflows/spectest.yaml:1]:
```suggestion
name: WebAssembly Core Specification Test
```

**Review by codefromthecrypt — APPROVED** [review]
Can you add a dimension for `REC-wasm-core-1` even though we don't have `WD-wasm-core-2` yet?

**Comment by mathetake** [comment]
you mean separate job or separate steps?

**Comment by codefromthecrypt** [comment]
sorry I meant spec versions * what we test them on

**Comment by mathetake** [comment]
ok lemme do in the separate PR where I am planning check-in v2 tests

**Comment by mathetake** [comment]
hmm well will add the dimension in this pr (without checking in()

**Comment by mathetake** [comment]
ok merging... the checking-in PR will follow soon

**Pull request merged**